### PR TITLE
PageMenu.jsm (follow up) - removal of redundant code, style clean up

### PR DESCRIPTION
**Pull request by janekptacijarabaci**
Removal of redundant code (e10s).

Style clean up:
https://hg.mozilla.org/mozilla-central/diff/239dd56733c4/toolkit/modules/PageMenu.jsm

---

I've created the new build (x32, Windows) and tested
(only: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu).

### CODE REVIEW

File: browser/modules/PageMenu.jsm [hunk 1]
```diff
@@ -13,13 +13,11 @@ PageMenu.prototype = {
...
   _popup: null,
 
-  // Only one of builder or browser will end up getting set.
+  // Only one of builder will end up getting set.
   _builder: null,
-  _browser: null,
 
   // Given a target node, get the context menu for it or its ancestor.
   getContextMenu: function(aTarget) {
-    let pageMenu = null;
```
Comment (wolfbeast) [inline browser/modules/PageMenu.jsm]: Why are you removing this?

Comment (janekptacijarabaci) [inline browser/modules/PageMenu.jsm]: It is not used in the function body. It is a local variable.

Comment (wolfbeast) [inline browser/modules/PageMenu.jsm]: You're wrong. It's used by the caller. if you remove this, then the previous context menu won't be cleared out.

Comment (wolfbeast) [inline browser/modules/PageMenu.jsm]: Oh right.. that's weird. I think that's a mistake and should not have the let keyword. pageMenu is used by the caller, I'm sure the intention is to clear that out.

**Pull request merged**